contracts/enable-kv/enable-kv.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+usage()
+{
+    echo "usage: enable-kv.sh -c <contract directory > [-u <URL to nodoes>]" 
+}
+
+while [ "$1" != "" ]; do
+    case $1 in
+        -c | --contracts-dir )  shift
+                       contracts_dir=$1
+                       ;;
+        -u | --url )   shift
+                       url=$1
+                       ;;
+        * )            usage
+                       exit 1
+    esac
+    shift
+done
+
+if [ "$contracts_dir" = "" ]; then
+   usage
+   exit 1
+fi
+
+if [ "$url" = "" ]; then
+    url=http://127.0.0.1:8888
+fi
+
+curl -X POST $url/v1/producer/schedule_protocol_feature_activations -d '{"protocol_features_to_activate": ["0ec7e080177b2c02b278d5088611686b49d739925a92d9bfcacd7fc6b74053bd"]}'
+sleep 1s
+cleos set contract eosio $contracts_dir/eosio.boot -p eosio@active
+sleep 1s
+#cleos set contract eosio $contracts_dir/eosio.bios -p eosio@active
+sleep 1s
+#cleos push action eosio setkvparams '[{"max_key_size":64, "max_value_size":1024, "max_iterators":128}]' -p eosio@active
```

Comment:
Remove commented out code. I forgot to remove them before publishing it,

Reply:
Done.